apps/masterbots.ai/components/layout/sidebar/sidebar-link.tsx

```
@@ -128,17 +139,21 @@ export default function SidebarLink({ category, isFilterMode, page }: SidebarLin
+      //  href={page === 'profile' ? `/u/${slug}/t/${toSlug(category.name)}` :`/c/${toSlug(category.name)}`}
          className={cn(
           'flex items-center p-2 cursor-pointer',
           isActive && 'bg-gray-200 dark:bg-mirage',
           page === 'profile' && 'pl-6'
         )}
-        onClick={handleClickCategory}
+        onClick={(e) => {
+          e.preventDefault();
+          handleClickCategory(e);
+        }}
       >
```

Biome (1.9.4)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)